.gitlab-ci.yml

```diff
@@ -250,12 +255,12 @@ run-certification-tests:
     # device to test with: prplmesh for dummy bwl, netgear-rax40 for dwpal on rax40
     DEVICE_UNDER_TEST: prplmesh
   script:
-      - |
-        if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
-            echo "Deploying to $DEVICE_UNDER_TEST"
-            tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
-        fi
-      - /easymesh_cert/run_test_file.sh -v -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
+    - |
+      if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
+          echo "Deploying to $DEVICE_UNDER_TEST"
+          tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
+      fi
+    - /easymesh_cert/run_test_file.sh -v -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
```

Comment:
Please do the formatting corrections in a separate commit. (We should probably run yamllint on all yaml files...)